nx/include/switch/services/acc.h

```diff
@@ -33,6 +35,15 @@ Result accountInitialize(void);
 void accountExit(void);
 Service* accountGetService(void);
 
+/// Get the total number of user profiles
+Result accountGetUserCount(s32* user_count);
+
+/**
+ * @brief Get a list of all user IDs
+ * @param userIDs Pointer to array of user IDs, array must be exactly size 8
```

Comment:
Replace 8 with ACC_USER_LIST_SIZE.

Does official user-process sw have this size hard-coded to this?